soc/bflb/bl60x/Kconfig

```diff
@@ -15,7 +15,6 @@ config SOC_SERIES_BL60X
 	select RISCV
 	select RISCV_HAS_CLIC
 	select RISCV_MACHINE_TIMER
-	select RISCV_SOC_HAS_CUSTOM_IRQ_HANDLING
```

Comment:
This should be in a separate commit. We are dropping this in favor of what ?

Reply:
separated the changes related to the interrupt cleanup in their own commit.